scripts/main.js

```diff
@@ -0,0 +1,348 @@
+"use strict";
+document.addEventListener("keydown", (event) => move(event.key));
+
+const rows = document.querySelectorAll("tr");
```

Comment:
The 'rows' variable is only used inside the 'drawBoard' function. Consider moving this declaration inside the function where it's used to limit its scope and avoid potential conflicts with other parts of the code.